### PR TITLE
RD2 Recurring Donation Page Layout Updates

### PR DESCRIPTION
**Pull request by sam-knox**
# Critical Changes

# Changes

# Issues Closed

# Community Ideas Delivered

# New Metadata

# Deleted Metadata

### CODE REVIEW

**Comment by sam-knox** [comment]
**lurch: attach W-040335

**Comment by LurchTheButler** [comment]
Tracking <a href="https://foundation.my.salesforce.com/a2x1E000001HtuMQAS" target="_blank">W-040335</a>

File: unpackaged/config/rd2_post_config/layouts/npe03__Recurring_Donation__c-Enhanced Recurring Donations Layout.layout [hunk 1]
```diff
@@ -2,91 +2,103 @@
...
             <layoutItems>
                 <behavior>Edit</behavior>
-                <field>%%%NAMESPACE%%%PaymentMethod__c</field>
-            </layoutItems>
-            <layoutItems>
-                <behavior>Required</behavior>
-                <field>npe03__Installment_Period__c</field>
+                <field>npe03__Installments__c</field>
             </layoutItems>
+            <layoutItems></layoutItems>
```
Comment (npsp-reedestockton) [inline unpackaged/config/rd2_post_config/layouts/npe03__Recurring_Donation__c-Enhanced Recurring Donations Layout.layout:63]: I learned something new here.  This needs to be:

``` 
           <layoutItems>
                <emptySpace>true</emptySpace>
            </layoutItems>
```

**Comment by sam-knox** [comment]
@rponti-sforg - This PR is ready for QA, if it even needs it.

**Pull request merged**